spec/features/listing_peeps_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+
+require_relative 'helpers/session'
+include SessionHelpers
+
+feature "Maker browses a list of peeps" do
+
+  before(:each) do
+
+    Maker.create( :name => "Snow White",
+                  :username => "red_apple",
+                  :email => "snow_white@example.com",
+                  :password => "seven_dwarfs",
+                  :password_confirmation => "seven_dwarfs")
+
+    sign_in('red_apple', 'seven_dwarfs')
+      add_peep( :message => "Hello Chitter!")
```

Comment:
these spaces are inconsistent - don't have a space after ( if you don't have one before the ending )